src/ngraph_encapsulate_op.cc

```diff
@@ -319,7 +319,8 @@ class NGraphEncapsulateOp : public OpKernel {
         m_ng_functions.erase(m_lru.back());
 
         // Call delete function here pf he erased func
-        op_backend->remove_compiled_function(evicted_ng_function);
+        auto exec = op_backend->compile(evicted_ng_function);
```

Comment:
Why do we need to compile to remove the function? Talk to the nGraph folks and see if there is an equivalent method.